src/SFA.DAS.EmployerFinance.Web/Startup/Startup.cs

```diff
@@ -1,44 +1,58 @@
 ﻿using System;
+using System.Globalization;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerFinance.Configuration;
+using SFA.DAS.EmployerFinance.Configuration.Extensions;
 using SFA.DAS.EmployerFinance.Startup;
+using SFA.DAS.EmployerFinance.Web.Authentication;
 using SFA.DAS.EmployerFinance.Web.DependencyResolution;
 using SFA.DAS.EmployerFinance.Web.Filters;
 using SFA.DAS.UnitOfWork.Mvc;
 using StructureMap;
 
-namespace SFA.DAS.EmployerFinance.Web
+namespace SFA.DAS.EmployerFinance.Web.Startup
 {
     public class Startup
```

Comment:
What do you think to renaming this to `AspNetStartup`?

Reply:
Personally, I think the AspNet prefix is unnecessary - you could conceivably add `AspNet` to all of the folders in the web project, as they all relate to asp.net, e.g. AspNetViews etc.!